src/main/java/net/praqma/tracey/cli/Tracey.java

```diff
@@ -38,6 +34,7 @@ public static void main(String[] args) throws Exception {
         sayParser.addArgument("-n", "--node").dest("node").help("The URL of the RabbitMQ server");
```

Comment:
Let's read my initial comment carefully - **please extend help message and state default values** that will be used in the argument is not provided.